src/components/images/ImgixImage.tsx

```diff
@@ -97,19 +97,27 @@ class ImgixImage extends React.PureComponent<MergedImgixImageProps, ImgixImagePr
 
     const Component = maybeComponent || (shouldUseFasterImage ? FasterImageView : FastImage);
 
-    const conditionalProps = shouldUseFasterImage
-      ? {
-          key: `${typeof source === 'object' && source.uri ? source.uri : ''}` || undefined,
-          onError: this.props.onError,
-          onLoad: undefined,
-          onSuccess: this.props.onLoad,
-        }
-      : {
-          key: `${typeof source === 'object' && source.uri ? source.uri : JSON.stringify(source)}-${retryCount}`,
-          onError: this.handleError,
-        };
-
-    return <Component {...props} {...conditionalProps} source={source} />;
+    if (shouldUseFasterImage) {
+      return (
+        <Component
+          {...props}
+          key={`${typeof source === 'object' && source.uri ? source.uri : ''}` || undefined}
+          onError={this.props.onError}
+          onLoad={undefined}
```

Comment:
this line can be removed

Reply:
i suppose overwriting onLoad to undefined was intended 🤔 doubtful but we could leave it to err on the side of safety